pkgs/development/python-modules/scikit-optimize/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
```

Comment:
```suggestion
, fetchFromGitHub
, isPy27
```

Reply:
Sorry, was intending to get back to this, was attending to something else.